consensus/istanbul/backend/api.go

```diff
@@ -151,36 +179,33 @@ var (
 	errNoBlockNumber           = errors.New("block number is not assigned")
 )
 
-// GetCouncil retrieves the list of authorized validators at the specified block.
-func (api *APIExtension) GetCouncil(number *rpc.BlockNumber) ([]common.Address, error) {
+// GetSnapshot retrieves the state snapshot at a given block. The namespace is klay.
+func (api *APIExtension) GetSnapshot(number *rpc.BlockNumber) (*Snapshot, error) {
```

Comment:
What about making a private function? instead of adding new APIExtension method.
```suggestion
func getSnapshot(chain consensus.ChainReader, istanbul *backend, number *rpc.BlockNumber) (*Snapshot, error) {
```

Reply:
Thanks for your review.
I will add a private function like what you and [blukat29](https://github.com/blukat29) recommended.

Reply:
Reflected in https://github.com/klaytn/klaytn/pull/1468/commits/8b3142b26d372e9e286c0a5495e53a1d4a7915b0.
Thank you.